mtp_api/apps/core/tests/test_dump_and_upload_for_ap.py

```diff
@@ -0,0 +1,47 @@
+from django.core.management import call_command
```

Comment:
as above, tiny thing about ordering of imports. i'm being pedantic, but i think consistency is more important than any specific rules over any other specific rules.